math/benches/fields/mersenne31.rs

```diff
@@ -0,0 +1,195 @@
+use std::hint::black_box;
+
+use criterion::Criterion;
+use lambdaworks_math::field::{element::FieldElement, fields::mersenne31::field::Mersenne31Field};
+use rand::random;
+
+pub type F = FieldElement<Mersenne31Field>;
+
+#[inline(never)]
+#[no_mangle]
+#[export_name = "util::rand_mersenne31_field_elements"]
```

Comment:
The [no_mangle] and other stuff is not necessary. Changes are similar to those in the PR for MiniGoldilocks